hack/shell-with-gopath.sh

```diff
@@ -17,6 +17,7 @@
 # If SHELL_WITH_GOPATH is 0 then do not change directories.
 [ "${SHELL_WITH_GOPATH}" = "0" ] && exec /bin/sh "${@}"
 
+export OLD_GOPATH="${GOPATH}"
```

Comment:
I don't see anything using this exported env var. Where is it used?

Reply:
Doh! That was something I added and then didn’t need. I’ll remove it. Good catch.

Reply:
Doh! That was something I added and then didn’t need. I’ll remove it. Good catch.